### PR TITLE
Add some Entities to GoMint

### PR DESCRIPTION
**Pull request by Kaooot**

### CODE REVIEW

File: gomint-server/src/main/java/io/gomint/server/entity/monster/EntityTropicalFish.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     *
+     * @param world The world in which this entity is in
+     */
+    public EntityTropicalFish( WorldAdapter world ) {
+        super( EntityType.PUFFERFISH, world );
+        this.initEntity();
+    }
+
+    /**
+     * Create new entity salmon for API
```
Comment (lukeeey) [inline gomint-server/src/main/java/io/gomint/server/entity/monster/EntityTropicalFish.java:23]: Should be entity tropical fish

**Review by lukeeey — APPROVED** [review]
Other than the one minor doc comment, looks good to me :)

File: gomint-server/src/main/java/io/gomint/server/entity/monster/EntityTropicalFish.java (hunk 1 above)
Comment (Kaooot) [reply gomint-server/src/main/java/io/gomint/server/entity/monster/EntityTropicalFish.java:23]: Yes xD I am sorry, but here: https://github.com/GoMint/GoMint/commit/277df223c83114a5ac954bb3b5354c9071263d14#diff-0d78afa2ec163f820f1c3a98dd9a73e4 is Standing "entity chicken", i think thats not so important :)

**Comment by Kaooot** [comment]
👍

**Pull request merged**